docs/new-architecture-library-intro.md

```diff
@@ -71,6 +91,22 @@ export default (codegenNativeComponent<NativeProps>(
 ): HostComponent<NativeProps>);
 ```
 
+##### Fabric Component: TypeScript Example
+
+```ts
+import type { ViewProps } from 'ViewPropTypes';
+import type { HostComponent } from 'react-native';
+const codegenNativeComponent = require('codegenNativeComponent');
```

Comment:
I was trying something like this spec in another project. I have a `MapView` component which is an external pod which is included by the app. Everything builds, but when I run the app I encounter this error:

```
error: Error: Unable to resolve module codegenNativeComponent from /Users/cipolleschi/Community/RNNewArchitectureApp/AwesomeApp/node_modules/map-view/js/MapViewNativeComponents.ts: codegenNativeComponent could not be found within the project or in these directories:
  node_modules

If you are sure the module exists, try these steps:
 1. Clear watchman watches: watchman watch-del-all
 2. Delete node_modules and run yarn install
 3. Reset Metro's cache: yarn start --reset-cache
 4. Remove the cache: rm -rf /tmp/metro-*
  1 | import type { ViewProps } from 'ViewPropTypes';
  2 | import type { HostComponent } from 'react-native';
> 3 | const codegenNativeComponent = require('codegenNativeComponent');
```

My full spec is [here](https://github.com/react-native-community/RNNewArchitectureApp/commit/37e1711528dee7cc2c30768b217eb670400f34a3). 

I have also added the `react-native-codegen` package and the babel plugins like this:
```js
plugins: [
    '@babel/plugin-proposal-class-properties',
    './node_modules/react-native/packages/babel-plugin-codegen'
]
```

Any idea about this problem @charlesbdudley?

<img src="https://user-images.githubusercontent.com/11162307/160657484-d69d2997-9fd1-46cd-a88f-64396ddb04c0.png" width="240" />

Reply:
Discussed offline. The snippet should be updated to use:
```
import codegenNativeComponent from 'react-native/Libraries/Utilities/codegenNativeComponent';
```